src/query/service/src/interpreters/interpreter_table_optimize.rs

```diff
@@ -112,6 +113,10 @@ impl Interpreter for OptimizeTableInterpreter {
             }
         }
 
+        if do_statistic {
+            table.statistic(self.ctx.clone()).await?;
```

Comment:
shall we return here after the statistic has been called?

Reply:
OK